sharpd/sharp_zebra.c

```diff
@@ -102,9 +104,13 @@ void sharp_install_routes_helper(struct prefix *p, vrf_id_t vrf_id,
 	} else
 		temp = ntohl(p->u.val32[3]);
 
+	/* Only use backup route/nexthops if present */
+	if (backup_nhg && (backup_nhg->nexthop == NULL))
```

Comment:
nit: `if (backup_nhg && !backup_nhg->nexthop)`

Reply:
Because of the way the sharp code passes bits and pieces of its 'globals' struct through these apis, I wanted to be as clear as possible about what was meant, that's all. I didn't think it was _bad_ to be checking carefully?

Reply:
Fair enough.